providers/tentacle.rb

```diff
@@ -94,7 +94,7 @@
     action :run
     cwd tentacle_install_location
     code <<-EOH
-      .\\Tentacle.exe new-certificate -e "#{cert_file}" --console
```

Comment:
The issue isn't with the cert being generated.  This step is very much self contained however the issue is that the tentacle is not configured if the service exists as you can see below.  This is where the generated cert is used / setup for the tentacle.  The uninstall should remove the service.  What version of the tentacle are you using?

https://github.com/cvent/octopus-deploy-cookbook/blob/master/providers/tentacle.rb#L113-L132